docs/doc/30-reference/30-sql/10-dml/dml-copy-into-table.md

```diff
@@ -38,19 +38,48 @@ internalStage ::= @<internal_stage_name>[/<path>]
 externalStage ::= @<external_stage_name>[/<path>]
 ```
 
-### externalLocation (for Amazon S3-like)
+### externalLocation
 
-```
-externalLocation (for Amazon S3) ::=
+AWS S3 compatible object storage services:
+
+```sql
+externalLocation ::=
   's3://<bucket>[/<path>]'
-  [ { CREDENTIALS = ( {  { AWS_KEY_ID = '<string>' AWS_SECRET_KEY = '<string>' } } ) } ]
+  CONNECTION = (
+        ENDPOINT_URL = 'https://<endpoint-URL>'
+        ACCESS_KEY_ID = '<your-access-key-ID>'
+        SECRET_ACCESS_KEY = '<your-secret-access-key>'
+        REGION = '<region-name>'
+        ENABLE_VIRTUAL_HOST_STYLE = true|false)
+```
+
+| Parameter                 	| Description                                                                 	| Required 	|
+|---------------------------	|-----------------------------------------------------------------------------	|----------	|
+| `s3://<bucket>/[<path>]`    	| External files located at the AWS S3 compatible object storage.             	| Optional 	|
+| ENDPOINT_URL              	| The bucket endpoint URL starting with "https://".                                  	| Optional 	|
+| ACCESS_KEY_ID             	| Your access key ID for connecting the AWS S3 compatible object storage.     	| Optional 	|
```

Comment:
Please note here: If `ACCESS_KEY_ID` is not provided, the databend will access the bucket as anonymous users.